packages/appstore/src/components/options/options-accounts.scss

```diff
@@ -0,0 +1,205 @@
+            align-items: center;
+            padding: 0.6rem;
+            gap: 1.6rem;
+            background: var(--general-main-1);
+            border-radius: 0.4rem;
+            margin-left: 1rem;
+            @include mobile {
+                width: 4rem;
+                padding: 0.3rem;
+            }
+        }
+        &--add-options-scroller {
+            position: absolute;
+            right: 0;
+            z-index: 999;
+            opacity: 0.9;
+            padding: 1.9rem;
+        }
+        &--platform {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding: 1.6rem 2.3rem 1.6rem 2.4rem;
+            width: 110.5rem;
+            height: 14.7rem;
+            background: var(--general-section-1);
+            border-radius: 0rem 0.6rem 0.6rem;
+            @include mobile {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                padding: 0.8rem 0 0;
+                width: 32.8rem;
+                height: 28rem;
+                left: 0;
+            }
+            &--divider {
+                width: 4rem;
+                height: 0.1rem;
+                background: $color-grey-4;
+                transform: rotate(90deg);
+            }
+        }
+        &--platform-with-applauncher {
+            background: var(--general-main-1);
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding: 1.6rem 2.3rem 1.6rem 2.4rem;
+            width: 100%;
+            border-radius: 0rem 0.6rem 0.6rem;
+            @include mobile {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                padding: 0.8rem 0 0;
+                width: 32.8rem;
+                height: 28rem;
+                left: 0;
+            }
+            &--divider {
+                width: 4rem;
+                height: 0.1rem;
+                background: $color-grey-4;
+                transform: rotate(90deg);
+            }
+        }
+        &--divider {
+            width: 96%;
+            height: 0.1rem;
+            background: $color-grey-4;
+            flex: none;
+            align-self: stretch;
+            flex-grow: 0;
+            margin-left: 2.4rem;
+            @include mobile {
+                margin-left: 1rem;
+            }
+        }
+    }
+
+    .dc-mobile-dialog__account-signup-mobile-dialog {
+        margin: 20px 15px 10px;
+    }
+}
+.options-container {
+    gap: 1.5rem;
+}
+.options-container-app-launcher {
+    gap: 0rem;
+    @include mobile {
+        gap: 1.5rem;
+    }
+}
+.dc-mobile-dialog__account-switcher-container {
+    margin: 20px 16px 0px 20px;
+    .dc-modal-footer {
+        position: fixed;
+        bottom: 0px;
+        width: 100%;
+    }
+}
+.options-accounts-container__loader {
+    width: 110.2rem;
+    height: 30.3rem;
+}
```

Comment:
why this add options button is like this ?
![image](https://user-images.githubusercontent.com/82078941/196022377-d2c2b5ff-d8ac-4b6a-b1db-19e8b9a2fa93.png)